src/App.tsx

```diff
@@ -50,15 +50,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && !isLoaded && <Loader />}
 
-                {author && loaded && hasError && (
+                {author && isLoaded && hasError && (
```

Comment:
we only get the error when the request is finished, so we don't need to check the status of the loader
```suggestion
                {(author && hasError) && (
```